automation/test.sh

```diff
@@ -1,3 +1,28 @@
+Skip to content
```

Comment:
@leyao-daily I think that was not intentinal.

Reply:
sorry. solved.